.circleci/config.yml

```diff
@@ -178,6 +178,23 @@ jobs:
             echo "$DOCKER_PASS" | docker login --username $DOCKER_USER --password-stdin
             docker build -t okteto/okteto:${CIRCLE_TAG} --build-arg VERSION_STRING=$CIRCLE_TAG .
             docker push okteto/okteto:${CIRCLE_TAG}
+      - run:
+          name: Auto-update-actions
+          command: |
+            curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo gpg --dearmor -o /usr/share/keyrings/githubcli-archive-keyring.gpg && sudo apt update && sudo apt install gh
+            gh auth login --with-token $GITHUB_TOKEN
+            curl -s https://api.github.com/repos/skyscanner/turbolift/releases/latest | grep "browser_download_url.*Linux_x86" | cut -d : -f 2,3 | tr -d \" | wget -O turbolift.tar -qi - && tar -xf turbolift.tar turbolift && mv turbolift /usr/local/bin/turbolift
+            chmod +x /usr/local/bin/turbolift
+            turbolift init --name update-actions-to-$CIRCLE_TAG
+            cd update-actions-to-$CIRCLE_TAG
+            echo "#Update action to $CIRCLE_TAG" > README.md
+            repos=("deploy-preview" "delete-namespace" "build" "destroy-preview" "deploy-stack" "namespace" "pipeline" "push" "destroy-pipeline" "create-namespace" "login" "apply")
+            printf "okteto/%s\n" "${repos[@]}" > repos.txt
+            turbolift clone
+            turbolift foreach -- sed -iE 's_FROM\ okteto\/okteto\:\.\*_FROM\ okteto\/okteto\:"$CIRCLE_TAG"_' Dockerfile
+            turbolift commit --message "Update to okteto/okteto:$CIRCLE_TAG"
+            turbolift create-prs
+            turbolift foreach gh pr merge pchico83:update-actions-to-$CIRCLE_TAG -d
```

Comment:
Could we directly commit to the main branch instead if creating a "fake" PR? It's more straightforward, we all get less emails, and it's easier to track when looking at the git logs. We already follow that pattern with the brew repo.

Reply:
Also, could we automate the process to create the github release of each action?